website_google_tag_manager/models/website.py

```diff
@@ -1,11 +1,12 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 ABF OSIELL <http://osiell.com>
+# Copyright 2018 Tecnativa - Cristina Martin R.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models, fields
+from odoo import fields, models
 
 
 class Website(models.Model):
     _inherit = 'website'
 
-    google_tag_manager_key = fields.Char(u"Google Tag Manager Key")
+    has_google_tag_manager = fields.Boolean('Google Tag Manager')
```

Comment:
Why adding this boolean? Isn't better to simply add the key if present? If not, you should program an onchange that unchecking this field, the field is emptied.

Reply:
It could be simpler, as in previous versions there was only the fields to include key_analytics, key_google maps, etc ..., but I have included that field to the same structure of website settings. Right now there isn't any group to include that field.

![captura de pantalla 2018-10-16 a las 10 12 47](https://user-images.githubusercontent.com/22169669/47002184-939b6100-d12c-11e8-92f2-3770845e702c.png)

Reply:
But you don't need an extra field for that. Use other structure.